utils/tests_fetcher.py

```diff
@@ -464,6 +464,7 @@ def module_to_test_file(module_fname):
     "tests/sagemaker/test_single_node_gpu.py",  # SageMaker test
     "tests/sagemaker/test_multi_node_model_parallel.py",  # SageMaker test
     "tests/sagemaker/test_multi_node_data_parallel.py",  # SageMaker test
+    "tests/mixed_int8/test_mixed_int8.py",  # SageMaker test
```

Comment:
Should not be `# SageMaker test` ..?

Reply:
Yes thank you for spotting this! Should be fixed in: 6d93424992ffdbfc57a8e944344a4d80633a97f1